app/views/reservations/_reservation_fields.html.haml

```diff
@@ -1,8 +1,15 @@
 .datetime-block
-  = f.input :reserve_start_date, :input_html => { :class => 'datepicker' }
+  - start_disabled = f.object.persisted? && !f.object.reserve_start_at_editable?
+  = f.input :reserve_start_date, :input_html => { :class => 'datepicker', disabled: start_disabled }
   .control-group
     .controls
       .string.optional.control-label &nbsp;
-      = time_select f, :reserve_start, minute_step: @instrument.reserve_interval
+      = time_select f, :reserve_start, { minute_step: @instrument.reserve_interval }, disabled: start_disabled
+      - if start_disabled
```

Comment:
Initially I tried to make it work without the hidden fields, but I was running into a issues because `Reservation#assign_times_from_params` will [set Reservation#reserve_start_at and Reservation#reserve_end_at to nil so the dates are set correctly](https://github.com/tablexi/nucore-open/blob/master/app/support/reservations/date_support.rb#L177-L186). Alternatively, I could probably fill in reserve_start_at and reserve_end_at in the controller if they don't exist in the params so we can remove the hidden fields from the page.